src/app/loginComponents/requests/requests.component.ts

```diff
@@ -118,4 +122,31 @@ export class RequestsComponent extends Base implements OnInit {
   rerequestReview(membership: OrganizationUser) {
     this.requestsService.requestRereview(membership.organization.id);
   }
+
+  removeOrganizationDialog(organizationName: string, organizationStatus: string, organizationID: number) {
+    const confirmationDialogData: ConfirmationDialogData = {
+      title: `Are you sure you wish to delete this ${organizationStatus} organization: ${organizationName}?`,
+      message: `All information associated with <b>${organizationName}</b> will be deleted.`,
+      cancelButtonText: 'Cancel',
+      confirmationButtonText: 'Delete'
+    };
+    this.confirmationDialogService
+      .openDialog(confirmationDialogData, bootstrap4mediumModalSize)
+      .pipe(takeUntil(this.ngUnsubscribe))
+      .subscribe(result => {
+        if (result) {
+          this.deleteOrganization(organizationID);
+        }
+      });
+  }
+
+  deleteOrganization(organizationID: number) {
+    this.userQuery.isAdminOrCurator$.pipe(takeUntil(this.ngUnsubscribe)).subscribe(isAdminOrCurator => {
+      if (isAdminOrCurator) {
```

Comment:
You could change this whole if/else block into one line, `this.requestsService.deleteOrganization(organizationID, isAdminOrCurator)`.

Reply:
Done.